controllers/users_controller.js

```diff
@@ -26,6 +26,25 @@ const {
 } = require("../utility/helpers");
 const { uploadImage, deleteImage } = require("../config/imageService");
 
+module.exports.userInfo = async (req, res) => {
+	let { uid, sortBy, sortType } = req.query;
+	let users;
+	if (uid) {
+		users = await User.findById(uid);
+	} else {
+		let role = ["core", "member"];
+		sortBy ? sortBy : "name";
+		sortType ? sortType : "asc";
+		users = await User.find(
+			{ role: { $in: role } },
```

Comment:
in this route,
- filter with showOnWebsite: true
- select fields name, email, designation, role, github, linkedin, twitter, portfolio, image